vendor/github.com/nsf/termbox-go/syscalls.go

```diff
@@ -0,0 +1,39 @@
+// +build ignore
+
+package termbox
+
+/*
+#include <termios.h>
+#include <sys/ioctl.h>
+*/
+import "C"
```

Comment:
um, cgo?

Reply:
I assure you that when you run `CGO_ENABLED=0 go build`, it generates a statically linked executable. I think this file is ignored, but dep still imports it.